src/create/index.js

```diff
@@ -20,6 +20,14 @@ const templates = glob(path.join(__dirname, "*.template")).map(f => path.parse(f
 module.exports = {
 	command: "create <module>",
 	describe: `Create a Moleculer module (${templates.join(",")})`,
```

Comment:
Are you planning to expand the command? Like add JS template service? If so I would suggest to change the description to something more explicit. `Create a new service boilerplate` or something like this.

AFAIK we don't use the term `module` anywhere. It might confuse the users.